src/Counter.sol

```diff
@@ -1,14 +1,30 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.13;
 
-contract Counter {
-    uint256 public number;
+library Counters {
```

Comment:
The counter is just the sample contract from foundry, or we will use the counter contract for any purpose on the project?

Reply:
Yes, we will use the counter contract for increment and decrement of mappings within the application